src/Api.Rest.Dtos/Data/GenericSearchConditionDto.cs

```diff
@@ -20,12 +26,23 @@ namespace Zeiss.PiWeb.Api.Rest.Dtos.Data
 	/// Klasse zum Parsen einer Filterzeichenkette für Messdaten.
 	/// </summary>
 	public class GenericSearchConditionDto
-	{ }
+	{
+		#region constants
+
+		internal const string ConditionFieldName = "condition";
+		internal const string ConditionsFieldName = "conditions";
+		internal const string AttributeFieldName = "attribute";
+		internal const string FieldNameFieldName = "fieldName";

```

Comment:
Since FiledNameFieldName sounds quite funny, my suggestion would be to rename this and the other to ...PropertyName instead, since the annotation for JSON is also JsonProperty/PropertyName. FieldNamePropertyName also has two times 'Name' but it is a little bit better. Or maybe only 'Property'?